src/main/java/com/dtolabs/rundeck/plugin/overthere/OTWinRMNodeExecutor.java

```diff
@@ -315,6 +331,17 @@ public static CmdLine buildCmdLineRaw(String... args) {
         return cmdLine;
     }
 
+    public static CmdLine buildCmdLinePowershell(String... args) {
+        CmdLine cmdLine = new CmdLine();
+        cmdLine.addRaw("powershell");
+        StringBuilder command = new StringBuilder("");
+        for (String s : args) {
+            command.append(s.replaceAll("\"", "\\\\\"")).append(" ");
```

Comment:
(this is replacing every double-quote with two-backslashes-followed-by-double-quote, agreed?)

Reply:
Agreed, otherwise quote are consumed by rundeck

Reply:
okay